examples/using-gatsby-image/src/pages/dominant-color.js

```diff
@@ -0,0 +1,113 @@
+import React from "react"
+import { graphql } from "gatsby"
+import { GatsbyImage, getImage } from "gatsby-plugin-image"
+import ImageGallery from "../components/image-gallery"
+import PageTitle from "../components/page-title"
+import Layout from "../components/layout"
+
+const DominantColor = ({ data, location }) => (
+  <Layout
+    location={location}
+    image={getImage(data.coverImage.localFile)}
+    imageTitle={`“${data.coverImage.title}” by ${data.coverImage.credit} (via unsplash.com)`}
+  >
+    <PageTitle>Dominant Color Placeholders</PageTitle>
+    <GatsbyImage
+      image={getImage(data.floatingImage.localFile)}
+      alt={`“${data.floatingImage.title}” by ${data.floatingImage.credit} (via unsplash.com)`}
+    />
+    <p>
+      The default Dominant color technique uses progressive loading to make a
+      fast, visually pleasing experience without waiting for a full-resolution
+      image with a blank screen.
+    </p>
+    <h2>Progressive Loading with Minimal Effort</h2>
+    <p>
+      The magic of Gatsby Image's Dominant color technique means that you can
+      load an image at moderate resolution and not have to bother with creating
```

Comment:
This sentence doesn't make sense to me (specifically the moderate resolution bit)